big-int/src/operators/multiply.cpp

```diff
@@ -35,17 +35,38 @@ namespace libbig
 {
     largeInt largeInt::operator*(const largeInt &next_number) {
 
-        auto append_zeroes = [](const largeInt &x, const int factor) {
-            int k;
+        using Iterator = std::string::reverse_iterator; 
```

Comment:
something like `std::string_reverse_iterator iter1` will be good, `using` directive is not recommended to use in this way

Reply:
I used that to shorten `std::string::reverse_iterator`, should i use
typedef instead?

Reply:
you could have used `auto` instead.

Reply:
Okay